src/sentry/testutils/silo.py

```diff
@@ -164,3 +166,69 @@ def test_something(self):
     """
     with override_settings(SILO_MODE=SiloMode.MONOLITH):
         yield
+
+
+def psql_function(
```

Comment:
Feel free to remove this from the PR -> i'm going to extract it for the foreign key extraction work I'm on.  I'll need to create a custom psql user in tests to validate deletions behave properly.